plugins/modules/junos_command.py

```diff
@@ -222,7 +222,7 @@ def rpc(module, items):
 
         if (
             name == "command"
-            and text.startswith("show configuration")
+            and text == "show configuration"
```

Comment:
should this be `text.srip()`?

Reply:
exact match here would avoid fetching all the config for filtered target config command.